ppr-ui/src/views/Dashboard.vue

```diff
@@ -431,12 +437,22 @@ export default class Dashboard extends Vue {
     return this.getSearchHistory?.length || 0
   }
 
-  private get hasPPRProduct (): boolean {
-    return this.getUserProductSubscriptionsCodes.includes(ProductCode.PPR)
+  private get hasPPR (): boolean {
+    // For Staff we check roles, for Client we check Products
+    if (this.isRoleStaff || this.isRoleStaffBcol || this.isRoleStaffReg) {
```

Comment:
For a consolidated staff check like this, is there already a getter? If not, perhaps we could create one.
Seems useful, saves some imports and duplication here if created in the store.

Reply:
There is no generic getter to check for all staff options. I could add one, as we use it in multiple places.

Reply:
I will add this getter in another PR. Just want to keep this one as a small fix.